app/javascript/dashboard/i18n/locale/en/teamsSettings.json

```diff
@@ -0,0 +1,123 @@
+{
+  "TEAMS_SETTINGS": {
+    "NEW_TEAM": "Create new team",
+    "HEADER": "Teams",
+    "SIDEBAR_TXT": "<p><b>Teams</b></p> <p>Organizing agents into teams let you use chatwoot the way your company is structured or to  segment the agents by their responsibilities. <br /> With Chatwoot you can create multiple teams and can have any number of agents, also a user can be part of multiple teams. <br/> You can assign conversations in chatwoot to a team so that it becomes a shared responsibility. </p>",
+    "LIST": {
+      "404": "There are no teams created on this account.",
+      "EDIT_TEAM": "Edit team"
+    },
+    "CREATE_FLOW": {
+      "CREATE": {
+        "TITLE": "Create a new team",
+        "DESC": "Add a title and description to your new team."
+      },
+      "AGENTS": {
+        "BUTTON_TEXT": "Add agents to team",
+        "TITLE": "Add agents to team - %{teamName}",
+        "DESC": "Add Agents to your newly created team. This lets you collaborate as a team on conversations, get notified on new events in the same conversation."
```

Comment:
Note: This needs to be changed, I will provide the updated text to be used here.